CHANGELOG.md

```diff
@@ -534,7 +534,7 @@ Migration Notes:
    // SELECT ... FROM Person p WHERE isJoe(p.name)
    // Becomes> SELECT ... FROM Person p WHERE 1 = isJoe(p.name)
    ```
-   This is because the aforementioned databases not not directly support boolean literals (i.e. true/false) or expressions
+   This is because the aforementioned databases do not directly support boolean literals (i.e. true/false) or expressions
```

Comment:
This is a notable change in that it wasn't the default change (which would have been to drop an instance of the doubled word).